app/models/concerns/user_access_key_overrides.rb

```diff
@@ -23,6 +23,13 @@ def password=(new_password)
     write_legacy_password_attributes(digest)
   end
 
+  def authenticatable_salt
+    return if encrypted_password_digest.blank?
+    Encryption::PasswordVerifier::PasswordDigest.parse_from_string(
+      encrypted_password_digest
+    ).password_salt
+  end
+
```

Comment:
Should we add a comment saying this is a Devise method required for authentication that we are overriding and that should not be removed as long as we need to override it?

Reply:
Done